api/management/commands/csv_ingest.py

```diff
@@ -1,21 +1,43 @@
 from django.core.management.base import BaseCommand
+from django.core import management
 import csv
+
 from api.models import Location, Hour
-import os
 
 
 class Command(BaseCommand):
-    help = 'Imports CSV file to populate local DB'
+    help = 'Ingests data into the db from csv_files. This command will kill and fill the db.'
 
     def handle(self, *args, **options):
-        for filename in ['community_gardens.csv']:
-            print('importing CSV file {0}'.format(filename))
-            with open(os.path.join('csv_files', filename), 'r') as f:
-                reader = csv.reader(f)
-                next(f)  # skips first line so we don't import headers
-                for r in reader:
-                    print('populating row: {0}'.format(r))
-                    loc = Location(name=r[1], category=r[2], address_1=r[3],
-                                   city=r[4], state=r[5], zipcode=r[6],
-                                   latitude=float(r[7]), longitude=float(r[8]))
-                    loc.save()
+        # this does a full kill and fill on the database.
+        management.call_command('flush', verbosity=0, interactive=False)
+        csv_files = ['farmers-market.csv',
+                     'community_gardens.csv', 'food-pantry.csv', 'supermarket.csv']
+        location_count = 0
+        for file in csv_files:
+            try:
+                with open('csv_files/{}'.format(file), 'r') as f:
+                    reader = csv.reader(f)
+                    next(f)
+                    for r in reader:
+                        loc = Location(name=r[0], address_1=r[1], address_2=r[2], city=r[3], state=r[
```

Comment:
Fixed this. I think my auto-formatter went a little crazy. I agree it's easier to read when the attributes are stacked up.